mlflow/store/rest_store.py

```diff
@@ -203,7 +204,7 @@ def get_metric_history(self, run_uuid, metric_key):
         """
         req_body = message_to_json(GetMetricHistory(run_uuid=run_uuid, metric_key=metric_key))
         response_proto = self._call_endpoint(GetMetricHistory, req_body)
-        return [Metric.from_proto(metric).value for metric in response_proto.metrics]
+        return [Metric.from_proto(metric) for metric in response_proto.metrics]
```

Comment:
This change seems appropriate, but should change the `return` documentation and should probably also be documented as part of the release notes.

Reply:
Ah yeah good catch

Reply:
This change fixes issue https://github.com/mlflow/mlflow/issues/1083
which was referenced by this PR https://github.com/mlflow/mlflow/pull/1084  that was tagged MLflow 1.0 on Apr 3